### PR TITLE
Fix missing buttons example

### PR DESCRIPTION
**Pull request by retlehs**
## Description

The buttons block is missing an example from the block inserter

[The button block has an example](https://github.com/WordPress/gutenberg/blob/11a3967416dec9e01a588350566cc7dfd2256a3b/packages/block-library/src/button/index.js#L26-L31), but it isn't used from the inserter

## How has this been tested?

I forked and `git clone`'d the repo, built the plugin locally and loaded on my local dev WP running 5.4.2 and no other plugins, default theme

### Before

Screen Shot 2020-07-28 at 12 41 05 PM

### After

Screen Shot 2020-07-28 at 12 45 00 PM

## Types of changes

Enhancement: This makes sure that the buttons block preview shows an example

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [ ] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

**Comment by Soean** [comment]
The buttons block contains inner blocks of the type `core/button`. If you want to preview custom buttons, you have to add inner blocks:

```JS
	example: {
		innerBlocks: [
			{
				name: 'core/button',
				attributes: {
					className: 'is-style-fill',
					backgroundColor: 'vivid-green-cyan',
					text: __( 'Call to Action' ),
				},
			},
		],
	},
```

If you want to show the example like in the screenshot, you just need to add an empty object:

```JS
	example: {},
```

**Comment by retlehs** [comment]
@Soean thank you! i updated this PR to just pass the empty `example` object and it still produces the same output as you mentioned

please let me know if there's any other changes i should make

**Review by Soean — APPROVED** [review]
I think it's good for the example preview. Thanks for contributing!

File: packages/block-library/src/buttons/index.js [hunk 1]
```diff
@@ -23,6 +23,7 @@ export const settings = {
 	),
 	icon,
 	keywords: [ __( 'link' ) ],
+	example: {},
```
Comment (youknowriad) [inline packages/block-library/src/buttons/index.js:26]: maybe we can be more creative and use something close to the two or three buttons pattern here.

**Pull request merged**